### DISCUSSION TITLE
打工魂 | /cn/2022/07/worker-soul/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1543] [Comments]
一冰箱的剩菜饭
欲请钟大仙
来随意聚个简餐
前后请了两遍
还是巍然不动如山
[&hellip;] 大仙回曰
这大周一的恐怕不便
大家上学的上学
上班的上班
我还急着赶论文死线
[&hellip;] 这听来着实可怜
我一声慨叹
独立日乃全国法定假日
你居然
还在披肝沥胆
[&hellip;] 大仙：哪尼
独立日　居然是今天
我：哪尼
居然还说居然
好呀么个　山中寒尽不知年
[&hellip;] 看　 &hellip;

https://yihui.org/cn/2022/07/worker-soul/

**Comment by ZaiyanHere** [comment]
哪尼
独立日　居然是今天
哪尼
居然还说居然
哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈好好笑 谢谢yihui贡献的笑点
